File: OPENAPI-REST-API/swagger-client/dart/lib/model/inline_response20026.dart
part of swagger.api;

class InlineResponse20026 {
  /* Confirmation message. */
  String text = null;
/* The support ticket ID created for tracking the migration. Use this with `/tickets/{id}` to check migration progress. */
  int ticket = null;

  InlineResponse20026();

  @override
  String toString() {
    return 'InlineResponse20026[text=$text, ticket=$ticket, ]';
  }

  InlineResponse20026.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    text = json['text'];
    ticket = json['ticket'];
  }

  Map<String, dynamic> toJson() {
    return {
      'text': text,
      'ticket': ticket
     };
  }

  static List<InlineResponse20026> listFromJson(List<dynamic> json) {
    return json == null ? new List<InlineResponse20026>() : json.map((value) => new InlineResponse20026.fromJson(value)).toList();
  }

  static Map<String, InlineResponse20026> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, InlineResponse20026>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new InlineResponse20026.fromJson(value));
    }
    return map;
  }
}

File: OPENAPI-REST-API/swagger-client/csharp-dotnet2/src/main/CsharpDotNet2/myadmin-client-csharp-dotnet2/Model/InlineResponse20026.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class InlineResponse20026 {
    /// <summary>
    /// Confirmation message.
    /// </summary>
    /// <value>Confirmation message.</value>
    [DataMember(Name="text", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "text")]
    public string Text { get; set; }

    /// <summary>
    /// The support ticket ID created for tracking the migration. Use this with `/tickets/{id}` to check migration progress.
    /// </summary>
    /// <value>The support ticket ID created for tracking the migration. Use this with `/tickets/{id}` to check migration progress.</value>
    [DataMember(Name="ticket", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "ticket")]
    public int? Ticket { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class InlineResponse20026 {\n");
      sb.Append("  Text: ").Append(Text).Append("\n");
      sb.Append("  Ticket: ").Append(Ticket).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}
